src/cpu/aarch64/acl_post_ops.hpp

```diff
@@ -0,0 +1,225 @@
+                post_ops_.entry_.clear();
+                for (int idx = 1; idx < post_ops.len(); ++idx) {
+                    // Construct empty entry then copy, so that we can check for failure
+                    post_ops_.entry_.emplace_back();
+                    CHECK(post_ops_.entry_.back().copy_from(
+                            post_ops.entry_[idx]));
+                }
+            } else {
+                // Nothing to fuse, just copy all post ops
+                CHECK(post_ops_.copy_from(post_ops));
+            }
+
+            return init(engine, dst_md);
+        }
+
+        bool has_sum() const { return acp_.sum_index >= 0; }
+
+        acl_post_ops_conf_t acp_;
+
+        // Vector of acl_binary_t::pd_t or acl_eltwise_fwd_t::pd_t to construct
+        // primitives for later
+        std::vector<std::shared_ptr<primitive_desc_t>> post_op_pds;
+
+    private:
+        post_ops_t post_ops_;
+    };
+
+    acl_post_ops_t(const pd_t *pd) : primitive_t(pd) {}
+
+    status_t init(engine_t *engine) override {
+
+        for (auto &post_op_pd : pd()->post_op_pds) {
+            if (post_op_pd->kind() == primitive_kind::binary) {
+                auto acl_binary_pd
+                        = std::dynamic_pointer_cast<acl_binary_t::pd_t>(
+                                post_op_pd);
+                if (!acl_binary_pd) return status::runtime_error;
+                auto acl_binary
+                        = std::make_unique<acl_binary_t>(acl_binary_pd.get());
+                post_op_primitives.push_back(std::move(acl_binary));
+            } else if (post_op_pd->kind() == primitive_kind::eltwise) {
+                auto acl_eltwise_pd
+                        = std::dynamic_pointer_cast<acl_eltwise_fwd_t::pd_t>(
+                                post_op_pd);
+                if (!acl_eltwise_pd) return status::runtime_error;
+                auto acl_eltwise = std::make_unique<acl_eltwise_fwd_t>(
+                        acl_eltwise_pd.get());
+                post_op_primitives.push_back(std::move(acl_eltwise));
+            } else {
+                // acl_post_ops_t::pd_t should only have binary or eltwise
+                // pd_ts, see pd_t::init for more details
+                return status::runtime_error;
+            }
+        }
+        return status::success;
+    }
+
+    status_t create_resource(
+            engine_t *engine, resource_mapper_t &mapper) const override {
+        for (const auto &post_op : post_op_primitives) {
+            CHECK(post_op->create_resource(engine, mapper));
+        }
+        return status::success;
+    }
+
+    bool has_sum() const { return pd()->has_sum(); }
+
+    status_t execute(const exec_ctx_t &ctx) const override {
+        return status_t::dnnl_unimplemented;
+    }
+
+    status_t execute(const exec_ctx_t &ctx, void *src) const {
+        return execute_forward(ctx, src);
+    }
+
+private:
+    status_t execute_forward(const exec_ctx_t &ctx, void *src) const;
```

Comment:
The primitive doesn't have a propagation kind
```suggestion
    status_t execute(const exec_ctx_t &ctx, void *src) const;
```